service/src/test/resources/ehr/mapper/diagnosticreport/observation/observation_compound_statement_with_data_absent_reason_and_interpretation_and_body_site_and_method.xml

```diff
@@ -26,23 +26,6 @@
     </ObservationStatement>
 </component>
             <component typeCode="COMP" contextConductionInd="true">
-    <NarrativeStatement classCode="OBS" moodCode="EVN">
```

Comment:
Is this NarrativeStatement supposed to disappear?

Reply:
Yep, this is the `observation.comment`. You'll see as part of this story this is now appended to the `observation.interpretation.text` in the "AGGREGATE COMMENT SET" NarrativeStatement below, so we don't need it output here (duplicating information)